libc-top-half/musl/src/thread/pthread_setcancelstate.c

```diff
@@ -2,10 +2,12 @@
 
 int __pthread_setcancelstate(int new, int *old)
 {
+	#ifdef _REENTRANT
```

Comment:
#ifdef should be in column zero I think.

Wrap in `__wasilibc_unmodified_upstream`